src/Components/Modals/CreateActivationKeyWizard.js

```diff
@@ -53,9 +43,9 @@ const CreateActivationKeyWizard = ({
     error,
     data,
   } = useSystemPurposeAttributes();
-  const { data: activationKeys } = useActivationKeys();
   const { addSuccessNotification, addErrorNotification } = useNotifications();
   const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
```

Comment:
Question: Is the intention to set a description that is not filled in to an empty string? Or null?

Reply:
Yes I did this way because it ensures the description is a defined value from the start, avoiding undefined errors when its used or displayed in the UI. it will avoid needing to check null or undefined in the code for a more controlled approach.

Reply:
Ack - when the API request is sent is an empty string mapped to null and/or unset from the request body? Otherwise this will default to a description of `''` instead of just not setting it.

I don't know what the intended behavior is - but my hunch would be that it should be omitted, instead of set as `''`